experiments/n2v/bsd68/careamics/README.md

```diff
@@ -22,19 +21,21 @@ config = create_n2v_configuration(
     num_epochs=50,
     masked_pixel_percentage=0.2,
     struct_n2v_axis="none",
+    model_params={
+        "num_channels_init": 32
+    },
+    optimizer_params={
```

Comment:
The `optimizer_params` and `lr_scheduler_params` do not exist in the signature of `create_n2v_configuration`:

https://github.com/CAREamics/careamics/blob/0fa617d7e27c61005274e433a3422be97bcadd7d/src/careamics/config/configuration_factories.py#L748

While it would probably make sense to add them, this is currently not the case. You should test the code right away when making modifications!

Reply:
Just to clarify further the `model_parameter` is a valid parameter, hence the original comment.

Opened an issue: https://github.com/CAREamics/careamics/issues/478